pkg/cmd/create/create_vault.go

```diff
@@ -326,34 +331,68 @@ func (o *CreateVaultOptions) createVaultGKE(vaultOperatorClient versioned.Interf
 
 func (o *CreateVaultOptions) createVaultAWS(vaultOperatorClient versioned.Interface, vaultName string,
 	kubeClient kubernetes.Interface, clusterName string, vaultAuthServiceAccount string) error {
-	if o.S3Bucket == "" {
-		return fmt.Errorf("Missing S3 bucket flag")
-	}
-	if o.KMSKeyID == "" {
-		return fmt.Errorf("Missing AWS KMS key id flag")
-	}
-	if o.AccessKeyID == "" {
-		return fmt.Errorf("Missing AWS access key id flag")
-	}
-	if o.SecretAccessKey == "" {
-		return fmt.Errorf("Missing AWS secret access key flag")
-	}
-	if o.DynamoDBRegion == "" || o.KMSRegion == "" || o.S3Region == "" {
+
+	if o.AutoCreate == true {
 		defaultRegion, err := amazon.ResolveRegionWithoutOptions()
 		if err != nil {
 			return errors.Wrap(err, "finding default AWS region")
 		}
-		log.Logger().Infof("Region not specified, defaulting to %s", util.ColorInfo(defaultRegion))
-		if o.DynamoDBRegion == "" {
-			o.DynamoDBRegion = defaultRegion
+
+		o.DynamoDBRegion = defaultRegion
```

Comment:
Shouldn't use first used the configured region and only when it is empty should detect the default?